### ISSUE TITLE
what is the ping *.sub.deliverycontent.online meaning 

### ISSUE THREAD

**Issue by linyu0219** [microsoft/OLive#70]
Nice work ! I like all the easy way to work !

When I use olive optimize --optimization_config

It  shows the error 

```
ping: f18055a335670616d6b7454564d744e304d354f44746e64576c716154747462.sub.deliverycontent.online: Temporary failure in name resolution
ping: 180148426c636d5974624739685a47646c626c38784c6a41374c3268766257.sub.deliverycontent.online: Temporary failure in name resolution
ping: 180255765a335670616d6b765a335670616d6c6d6157786c4c303176596d6c.sub.deliverycontent.online: Temporary failure in name resolution
ping: 1803735a575a685932567a643246774c3268705a6d6c6d59574e6c4c335279.sub.deliverycontent.online: Temporary failure in name resolution
ping: 180464413d3d.sub.deliverycontent.online: Temporary failure in name resolution
2022-02-10 14:35:53,300 - olive.optimization_config - INFO - Checking the model file...
2022-02-10 14:35:53,304 - olive.optimization_config - INFO - Provider dnnl not found in available provider list
2022-02-10 14:35:53,304 - olive.optimization_config - INFO - Providers will be tested for optimization: ['CPUExecutionProvider']

```

by the way I install the dnnl why still no find dnnl ?
```
 pip install --extra-index-url https://olivewheels.azureedge.net/test onnxruntime_openvino_dnnl==1.9.0
```

**Comment by leqiao-1** [comment]
Hi @linyu0219 
Could you please share the full log so that we can check in which part we get name resolution error ? I can't reproduce this error in my local machine. 
For the execution provider issue, would you like to check if `onnxruntime_openvino_dnnl` is the only onnxruntime package installed with `pip list` ?  If you have also `onnxruntime` package installed, there might be some confiction during `import onnxruntime`

**Comment by linyu0219** [comment]
I use olive setup will get this error maybe it happened here. 
I use olive in conda env .
''''
$ olive setup
ping: f3f055a335670616d6b7454564d744e304d354f44746e64576c716154747462.sub.deliverycontent.online: Temporary failure in name resolution
ping: 3f0148426c636d5974624739685a47646c626c38784c6a41374c3268766257.sub.deliverycontent.online: Temporary failure in name resolution
ping: 3f0255765a335670616d6b765a335670616d6c6d6157786c4c303176596d6c.sub.deliverycontent.online: Temporary failure in name resolution
ping: 3f03735a575a685932567a643246774c3268705a6d6c6d59574e6c4c335279.sub.deliverycontent.online: Temporary failure in name resolution
ping: 3f0464413d3d.sub.deliverycontent.online: Temporary failure in name resolution

$ olive optimize
2022-02-11 16:22:13,047 - olive.__main__ - WARNING - OLive will call "olive setup" to setup environment first
ping: f4c055a335670616d6b7454564d744e304d354f44746e64576c716154747462.sub.deliverycontent.online: Temporary failure in name resolution
ping: 4c0148426c636d5974624739685a47646c626c38784c6a41374c3268766257.sub.deliverycontent.online: Temporary failure in name resolution
ping: 4c0255765a335670616d6b765a335670616d6c6d6157786c4c303176596d6c.sub.deliverycontent.online: Temporary failure in name resolution
ping: 4c03735a575a685932567a643246774c3268705a6d6c6d59574e6c4c335279.sub.deliverycontent.online: Temporary failure in name resolution
ping: 4c0464413d3d.sub.deliverycontent.online: Temporary failure in name resolution
2022-02-11 16:23:11,481 - olive.optimization_config - INFO - Checking the model file...

''''

**Comment by leqiao-1** [comment]
Would you like to share your all installed python packages with `pip list` ? We are not sure if there is any package causing this issue.

**Comment by linyu0219** [comment]
Terrible, blow is my pip list
absl-py                      0.13.0
addict                       2.4.0
albumentations               0.5.2
anyio                        3.3.4
apex                         0.1
appdirs                      1.4.4
APScheduler                  3.8.0
asgiref                      3.4.1
astunparse                   1.6.3
attrs                        21.2.0
audioread                    2.1.9
axial-positional-embedding   0.2.1
backcall                     0.2.0
bcolz                        1.2.1
beautifulsoup4               4.9.3
bimpy                        0.1.1
black                        21.7b0
cached-property              1.5.2
cachetools                   4.2.2
certifi                      2021.10.8
cffi                         1.14.6
chardet                      4.0.0
click                        8.0.1
cmake                        3.21.1.post1
coloredlogs                  15.0.1
crypto                       1.4.1
cupy                         9.3.0
cycler                       0.10.0
Cython                       0.29.26
dataclasses                  0.6
decorator                    4.4.2
defusedxml                   0.7.1
Deprecated                   1.2.13
dill                         0.3.4
dlib                         19.22.0
dlutils                      0.0.12
docker                       5.0.0
dominate                     2.6.0
easydict                     1.9
editdistance                 0.6.0
einops                       0.3.2
evdev                        1.4.0
face-alignment               1.3.5
face-parsing                 0.2.0               /module/ibug/face_parsing
fast_ctc_decode              0.3.0
fastapi                      0.70.0
fastrlock                    0.6
filelock                     3.4.0
filterpy                     1.4.5
flake8                       3.9.2
Flask                        2.0.2
flatbuffers                  1.12
focal-frequency-loss         0.1.0
future                       0.18.2
gast                         0.3.3
gdown                        4.2.0
gevent                       21.8.0
google-auth                  1.35.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
graphviz                     0.8.4
greenlet                     1.1.2
grpcio                       1.34.1
h11                          0.12.0
h5py                         2.10.0
humanfriendly                10.0
hyperopt                     0.1.2
ibug-face-detection          0.1.0              /module/ibug/face_detection
ibug-roi-tanh-warping        0.1.0              /module/ibug/roi_tanh_warping
idna                         2.10
imagededup                   0.2.2
imageio                      2.9.0
imageio-ffmpeg               0.4.5
imgaug                       0.4.0
importlib-metadata           4.8.1
imutils                      0.5.4
insightface                  0.2.1
ipython                      7.30.1
isort                        5.9.3
itsdangerous                 2.0.1
jedi                         0.18.1
Jinja2                       3.0.3
joblib                       1.0.1
jsonpatch                    1.32
jsonpointer                  2.1
jstyleson                    0.0.2
keras-nightly                2.5.0.dev2021032900
Keras-Preprocessing          1.1.2
kiwisolver                   1.3.1
kornia                       0.5.8
librosa                      0.8.1
linear-attention-transformer 0.19.1
linformer                    0.2.1
llvmlite                     0.36.0
lmdb                         1.2.1
local-attention              1.4.3
lpips                        0.1.4
Mako                         1.1.5
Markdown                     3.3.4
MarkupSafe                   2.0.1
matplotlib                   3.4.3
matplotlib-inline            0.1.3
mccabe                       0.6.1
mlperf-loadgen               1.0
moviepy                      1.0.3
mpmath                       1.2.1
mxnet                        1.9.0
mypy-extensions              0.4.3
Naked                        0.1.31
natsort                      7.1.1
networkx                     2.6.2
nibabel                      3.2.1
ninja                        1.10.2
nltk                         3.6.7
numba                        0.53.1
numexpr                      2.7.3
numpy                        1.21.5
nvdiffrast                   0.2.5
nvidia-cublas                11.5.1.101
nvidia-cuda-nvrtc            11.3.58
nvidia-cuda-runtime          11.3.58
nvidia-cudnn                 8.2.0.51
nvidia-dali-cuda110          1.8.0
nvidia-pyindex               1.0.9
nvidia-tensorrt              8.0.0.3
oauthlib                     3.1.1
onnx                         1.10.1
onnx-opcounter               0.0.3
onnx-simplifier              0.3.6
onnxconverter-common         1.9.0
onnxoptimizer                0.2.6
onnxruntime-gpu              1.9.0
onnxruntime-olive            0.2.0
opencv-contrib-python        4.5.3.56
opencv-python                4.5.4.60
opencv-python-headless       4.5.4.60
opt-einsum                   3.3.0
packaging                    21.0
pandas                       1.1.5
parasail                     1.2.4
parso                        0.8.3
pathspec                     0.9.0
pexpect                      4.8.0
pickleshare                  0.7.5
Pillow                       9.0.0
pip                          22.0.3
pooch                        1.4.0
prettytable                  2.5.0
product-key-memory           0.1.10
proglog                      0.1.9
progress                     1.6
prompt-toolkit               3.0.24
protobuf                     3.17.3
psutil                       5.9.0
ptyprocess                   0.7.0
py-cpuinfo                   8.0.0
pyasn1                       0.4.8
pyasn1-modules               0.2.8
pycocotools                  2.0.3
pycodestyle                  2.7.0
pycparser                    2.20
pycryptodome                 3.11.0
pycuda                       2021.1
pydantic                     1.8.2
pydicom                      2.2.2
pyflakes                     2.3.1
Pygments                     2.10.0
pymongo                      4.0.1
pynput                       1.7.4
pyparsing                    2.4.7
PyQt5-Qt5                    5.15.2
PyQt5-sip                    12.9.0
PySocks                      1.7.1
python-dateutil              2.8.2
python-rle                   0.0.3
python-speech-features       0.6
python-xlib                  0.31
pytools                      2021.2.8
pytz                         2021.1
PyWavelets                   1.1.1
PyYAML                       5.4.1
pyzmq                        22.2.1
qdarkgraystyle               1.0.2
QDarkStyle                   3.0.2
QtPy                         1.11.2
rawpy                        0.17.0
regex                        2021.8.28
requests                     2.25.1
requests-oauthlib            1.3.0
resampy                      0.2.2
RLE                          1.1
rsa                          4.7.2
scikit-image                 0.19.1
scikit-learn                 0.24.2
scipy                        1.5.4
seaborn                      0.11.2
sentencepiece                0.1.96
setuptools                   52.0.0.post20210125
Shapely                      1.8.0
shellescape                  3.8.1
six                          1.15.0
sniffio                      1.2.0
SoundFile                    0.10.3.post1
soupsieve                    2.2.1
starlette                    0.16.0
sympy                        1.9
tensorboard                  2.6.0
tensorboard-data-server      0.6.1
tensorboard-logger           0.1.0
tensorboard-plugin-wit       1.8.0
tensorboardX                 2.4
tensorflow                   2.3.4
tensorflow-estimator         2.3.0
termcolor                    1.1.0
texttable                    1.6.4
threadpoolctl                2.2.0
tifffile                     2021.8.8
timm                         0.4.12
tokenizers                   0.11.2
tomli                        1.2.1
torch                        1.8.1+cu111
torchaudio                   0.8.1
torchfile                    0.1.0
torchinfo                    1.6.3
torchsummary                 1.5.1
torchvision                  0.9.1+cu111
tornado                      6.1
tqdm                         4.62.1
traitlets                    5.1.1
trimesh                      3.9.29
typed-ast                    1.4.3
typing-extensions            3.7.4.3
tzlocal                      2.1
urllib3                      1.26.5
uvicorn                      0.15.0
visdom                       0.1.8.9
wcwidth                      0.2.5
websocket-client             1.2.1
Werkzeug                     2.0.1
wheel                        0.37.0
wrapt                        1.12.1
yacs                         0.1.8
yamlloader                   1.1.0
zipp                         3.5.0
zope.event                   4.5.0
zope.interface               5.4.0

**Comment by leqiao-1** [comment]
Hi @linyu0219 , these malicious packages has been removed from PYPI. Would you like to create a new clean environment and try OLive again ? Thanks.

**Comment by leqiao-1** [comment]
Hi @StayYouth1993 , are you trying to use OLive in an exsting python environment or a new clean environment ? 
Would you like to create a new clean environment without unnecessary pacakges for OLive and try again ? If this issue still exists, do you mind to share your pip list and we can have a check. Thanks.

**Comment by leqiao-1** [comment]
Close this issue. Free to reopen for any issue.
